test/e2e/ginkgo/README.md

```diff
@@ -29,7 +29,13 @@ To execute these tests, issue the following command. This will make a local imag
 KFLEX_DISABLE_CHATTY=true ginkgo --vv --trace --no-color
 ```
 
-To test the latest release image, pass `--released` to the test. For example:
+To pass additional command line flags/args to the embedded usage of the [setup-kubestellar.sh script](../common/setup-kubestellar.sh), pass one additional flag to the test suite, where this outer flag's name is `kubestellar-setup-flags` and the outer flag's value is the space-separated concatenation of all the additional setup-kubestellar flags/args. Remember that test suite args/flags must be preceeded on the `ginkgo` command line by `--`. Following is an example.
+
+```shell
+KFLEX_DISABLE_CHATTY=true ginkgo --vv --trace --no-color -- -kubestellar-setup-flags="--kubestellar-controller-manager-verbosity 5" 
+```
+
+To test the latest release image, either (a) pass the `--released` flag to setup-kubestellar using the technique above or (b) pass `--released` to the test suite. For example:
```

Comment:
To simplify, it might be worth while *in the future* to retire the released flag and switch to using kubestellar-setup-flags.

Reply:
Yes.